server/status.go

```diff
@@ -155,6 +165,65 @@ func (s *statusServer) handleLocalStatus(w http.ResponseWriter, r *http.Request,
 	w.Write(b)
 }
 
+// handleLocalLogs handles GET requests for list of available logs.
+func (s *statusServer) handleLocalLogs(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	log.Flush()
+	logFiles, err := log.ListLogFiles()
+	if err != nil {
+		log.Error(err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	b, contentType, err := util.MarshalResponse(r, logFiles, []util.EncodingType{util.JSONEncoding})
```

Comment:
It is unsafe to output an array in json without wrapping that array in a dict (https://github.com/tornadoweb/tornado/blob/f5380632c2c6c357eb5ec62e5ae3f416f56ba9f7/tornado/web.py#L683-L687). It's a minor issue here but we may want to enforce this in MarshalResponse.